<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bridgeEditorImportDialog</class>
 <widget class="QDialog" name="bridgeEditorImportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import User Plot Data . . .</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" columnstretch="10,0">
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="3">
      <widget class="QSpinBox" name="sbLines2Discard">
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="3">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Number of Header Lines to Discard:</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0" colspan="5">
      <widget class="QTableWidget" name="previewTable">
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
      </widget>
     </item>
     <item row="9" column="3" colspan="2">
      <widget class="QComboBox" name="cboCol3">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="3" colspan="2">
      <widget class="QComboBox" name="cboCol2">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="12" column="0" colspan="5">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>15</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0" colspan="5">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <widget class="QToolButton" name="btnBrowse">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="5">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Delimited File</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0" colspan="5">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Preview (first 10 rows):</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="5">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Header Rows</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="5">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>15</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="10" column="0" colspan="3">
      <widget class="QLabel" name="labelCol4">
       <property name="text">
        <string>Column 4</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="4">
      <widget class="QLineEdit" name="inFile"/>
     </item>
     <item row="2" column="0" colspan="5">
      <widget class="QGroupBox" name="grouBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Delimited Format</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="3">
         <widget class="QRadioButton" name="rbOther">
          <property name="text">
           <string>Other:</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">bgDelimiter</string>
          </attribute>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QRadioButton" name="rbTab">
          <property name="text">
           <string>Tab</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">bgDelimiter</string>
          </attribute>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="rbSpace">
          <property name="text">
           <string>Space</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">bgDelimiter</string>
          </attribute>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QRadioButton" name="rbCSV">
          <property name="text">
           <string>CSV</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">bgDelimiter</string>
          </attribute>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLineEdit" name="delimiter"/>
        </item>
       </layout>
      </widget>
     </item>
     <item row="11" column="0" colspan="3">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Null Value (optional)</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="3">
      <widget class="QLabel" name="labelCol2">
       <property name="text">
        <string>Column 2</string>
       </property>
      </widget>
     </item>
     <item row="7" column="3" colspan="2">
      <widget class="QComboBox" name="cboCol1">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="3">
      <widget class="QLabel" name="labelCol3">
       <property name="text">
        <string>Column 3</string>
       </property>
      </widget>
     </item>
     <item row="11" column="3" colspan="2">
      <widget class="QLineEdit" name="nullValue"/>
     </item>
     <item row="7" column="0" colspan="3">
      <widget class="QLabel" name="labelCol1">
       <property name="text">
        <string>Column 1</string>
       </property>
      </widget>
     </item>
     <item row="10" column="3" colspan="2">
      <widget class="QComboBox" name="cboCol4">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbOk">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="bgDelimiter"/>
 </buttongroups>
</ui>
